docs/subject.yaml

```
@@ -119,4 +119,65 @@ paths:
+          name: id
+          required: true
+          description: The unique identifier of the subject
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful response — subject found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Subject' 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)